src/cli.yml

```diff
@@ -0,0 +1,15 @@
+name: wiki
+version: "0.1"
```

Comment:
This is not needed and can be replaced within the source with the `crate_verstion!()` macro or the macro app_from_crate!: Combines crate_version!, crate_name!, crate_description!, and crate_authors! into a single macro call to build a default App instance from the Cargo.toml fields